building/components/homeworld-rkt/glass.yaml

```diff
@@ -68,14 +68,21 @@ build:
 
       ./autogen.sh
 
+      if [[ -v CI ]]
+      then
+        DEFAULT_FLAVOR=coreos
+      else
+        DEFAULT_FLAVOR=kvm
+      fi
+
       ./configure \
           --disable-tpm --prefix=/usr \
           --with-stage1-flavors=coreos,kvm \
-          --with-stage1-default-flavor=kvm \
+          --with-stage1-default-flavor=${DEFAULT_FLAVOR} \
           --with-coreos-local-pxe-image-path=coreos_production_pxe_image.cpio.gz \
           --with-coreos-local-pxe-image-systemd-version=v233 \
           --with-stage1-default-images-directory=/usr/lib/rkt/stage1-images \
-          --with-stage1-default-location=/usr/lib/rkt/stage1-images/stage1-kvm.aci
+          --with-stage1-default-location=/usr/lib/rkt/stage1-images/stage1-${DEFAULT_FLAVOR}.aci
```

Comment:
Honestly, we may as well just set the default to coreos in general -- it doesn't really make sense for us to worry about using kvm right now, because of how early we are in development.

(I was going to make that change anyway, and you may as well just roll it in here and not need the CI gating.)